yarn.lock

```diff
@@ -2293,40 +2441,75 @@
+"@polkadot/api-derive@1.11.2":
+  version "1.11.2"
+  resolved "https://registry.yarnpkg.com/@polkadot/api-derive/-/api-derive-1.11.2.tgz#e9b01b0f08aa25ec7842973b4c884d6b74996bb6"
+  integrity sha512-jwU8oZItbJWKOIF/mJDTjkGV2t8Fz45ZR5WbPPWid7ojzPyswQx+qepVco8j1lGvFs4i+kKG4B01OCJ2KErTzA==
   dependencies:
     "@babel/runtime" "^7.9.2"
-    "@polkadot/api" "1.9.1"
-    "@polkadot/rpc-core" "1.9.1"
-    "@polkadot/rpc-provider" "1.9.1"
-    "@polkadot/types" "1.9.1"
-    "@polkadot/util" "^2.7.1"
-    "@polkadot/util-crypto" "^2.7.1"
+    "@polkadot/api" "1.11.2"
+    "@polkadot/rpc-core" "1.11.2"
+    "@polkadot/rpc-provider" "1.11.2"
+    "@polkadot/types" "1.11.2"
+    "@polkadot/util" "^2.8.1"
+    "@polkadot/util-crypto" "^2.8.1"
     bn.js "^5.1.1"
     memoizee "^0.4.14"
-    rxjs "^6.5.4"
+    rxjs "^6.5.5"
 
-"@polkadot/api@1.9.1", "@polkadot/api@^1.9.1":
-  version "1.9.1"
-  resolved "https://registry.yarnpkg.com/@polkadot/api/-/api-1.9.1.tgz#10956c3c07bf6ba132f94722bd4bc23d536a7296"
-  integrity sha512-dJlWvpilZCVdRgZ2HsG6fwkNGuHomfqHfjMMPPz1yLdnMybWpsWYxK5cYfYZ3Y9EI0KSv0W+ZU0o6LnQyE/lAA==
+"@polkadot/api-derive@1.12.0-beta.10":
+  version "1.12.0-beta.10"
+  resolved "https://registry.yarnpkg.com/@polkadot/api-derive/-/api-derive-1.12.0-beta.10.tgz#47ced1ba4fba7e835923c158e313175e38fbd3e0"
+  integrity sha512-PERoNueQaonZ0h3sJ/OVLoIDmOBo/D0VWqQU3vK1882a4kQKxPur5vAP41n//2BHFR6tGidJfiB/3/lnYH2udA==
+  dependencies:
+    "@babel/runtime" "^7.9.2"
+    "@polkadot/api" "1.12.0-beta.10"
+    "@polkadot/rpc-core" "1.12.0-beta.10"
+    "@polkadot/rpc-provider" "1.12.0-beta.10"
+    "@polkadot/types" "1.12.0-beta.10"
+    "@polkadot/util" "^2.9.0-beta.1"
+    "@polkadot/util-crypto" "^2.9.0-beta.1"
+    bn.js "^5.1.1"
+    memoizee "^0.4.14"
+    rxjs "^6.5.5"
+
+"@polkadot/api@1.11.2", "@polkadot/api@^1.9.1":
+  version "1.11.2"
+  resolved "https://registry.yarnpkg.com/@polkadot/api/-/api-1.11.2.tgz#f039af0ac06f7a0744e8f7b9d278f85ae2c7c0bf"
+  integrity sha512-SAuPPtEXfCMl3yCGN22sJmuiWKR0epU79KRKEqpp6pvZ2fMw7lIg1pW6KYn/bhaQUSEsS0cUJVSfz7UjKanA2w==
   dependencies:
     "@babel/runtime" "^7.9.2"
-    "@polkadot/api-derive" "1.9.1"
-    "@polkadot/keyring" "^2.7.1"
-    "@polkadot/metadata" "1.9.1"
-    "@polkadot/rpc-core" "1.9.1"
-    "@polkadot/rpc-provider" "1.9.1"
-    "@polkadot/types" "1.9.1"
-    "@polkadot/types-known" "1.9.1"
-    "@polkadot/util" "^2.7.1"
-    "@polkadot/util-crypto" "^2.7.1"
+    "@polkadot/api-derive" "1.11.2"
+    "@polkadot/keyring" "^2.8.1"
+    "@polkadot/metadata" "1.11.2"
+    "@polkadot/rpc-core" "1.11.2"
+    "@polkadot/rpc-provider" "1.11.2"
+    "@polkadot/types" "1.11.2"
+    "@polkadot/types-known" "1.11.2"
+    "@polkadot/util" "^2.8.1"
+    "@polkadot/util-crypto" "^2.8.1"
     bn.js "^5.1.1"
     eventemitter3 "^4.0.0"
-    rxjs "^6.5.4"
+    rxjs "^6.5.5"
+
+"@polkadot/api@1.12.0-beta.10", "@polkadot/api@^1.12.0-beta.10":
```

Comment:
There are two versions of api in node_modules, would be good to use 1.12-beta everywhere

Reply:
Should be better now, there's only `@polkadot/api@1.12.0-beta.10`